lib/ansible/module_utils/k8s/common.py

```diff
@@ -76,6 +76,10 @@
         'default': 'v1',
         'aliases': ['api', 'version'],
     },
+    'labels': {
```

Comment:
I don't think `labels` belongs in the common arg spec, it doesn't mean anything outside of `k8s_facts`